addons/godot-firebase/database/reference.gd

```diff
@@ -133,11 +130,11 @@ func _get_filter():
     # At the moment, this means you can't dynamically change your filter; I think it's okay to specify that in the rules.
     if !_cached_filter:
         _cached_filter = ""
```

Comment:
Can you add a variable for get_parent() here? That way, we don't have to have Godot traverse the tree every time it's called in this function.